test_cli.py

```diff
@@ -1087,6 +1088,56 @@ def test(ctx):
     check_contains(group_buuid[:8], run_command([cl, 'search', 'group={}'.format(group_name)]))
 
 
+@TestModule.register('search_time')
+def test(ctx):
+    name = random_name()
+    time1 = datetime.now().isoformat()
+    uuid1 = run_command([cl, 'run', 'date', '-n', name])
+    time.sleep(1)
```

Comment:
Couldn't make these tests stable without a sleep. Any suggestions? I saw other tests use sleeps so I assume this is okay, but any alternatives would be welcome.

Reply:
Is this because you need for the run to finish? I think there is a way to explicitly wait for the bundle to finish if that's what you need, rather than sleeping. Ideally we'd know why the test is flaky without the sleep, identify the explicit condition it needs to wait for, and wait for that specific condition rather than sleeping.

Reply:
I did not know I needed to wait for it to finish. That might be a better solution if run date takes long enough for that to make a difference. Do you know what that flag is? 
What I was seeing is that the run date completed so quickly the times weren't distinct enough for the comparisons to always work.

Reply:
I see. If we need sleep so the timestamps for different runs are actually different, then using sleep seems perfectly fine to me. And we shouldn't need to wait for the runs to complete in that case.

(for info, the test script issues commands to an externally running codalab server so commands like `run` etc are not synchronous by default)

Reply:
For `time.sleep(1)`, Is it because of the time used to search result is longer than usual?

Reply:
Document why you need the sleep.

Reply:
Will add comment